projects/ng-components/scripts/css-bundle.ts

```diff
@@ -2,20 +2,17 @@ const pathLib = require('path');
 const scssBundler = require('scss-bundle');
 const fs = require('fs-extra');
 
-
 /** Bundles all SCSS files into a single file */
 async function bundleScss(): Promise<void> {
   //console.log('running bundleScss');
+  const isolatedStyles = ['modal.css'];
   const bundler = new scssBundler.Bundler();
   const sourcePath = './projects/ng-components/src/_styles.css';
   const { found, bundledContent, imports } = await bundler.bundle(sourcePath, ['./src/**/*.css', './src/**/*.scss']);
```

Comment:
How come we did not combine the isolatedStyles with the rest of them. Just curious I dont know what I am talking about here.

Reply:
This temporary, but the idea is to expose isolates css files per component for those who don't want to inherit all the libraries css just to use a modal.

Reply:
Ahhh nice I get it. I suppose its just to guide you through like the basic stuff so you know what you added to your project. This is the main reason that reusable componetnts are either framework independent or have multiple style flavors (per fw)